homeassistant/generated/dhcp.py

```diff
@@ -276,6 +276,18 @@
         "hostname": "polisy*",
         "macaddress": "000DB9*",
     },
+    {
+        "domain": "lamarzocco",
+        "hostname": "gs[0123456789][0123456789][0123456789][0123456789][0123456789][0123456789]",
```

Comment:
```suggestion
        "hostname": "gs[0-9]{6}",
```

Reply:
This are glob pattern not regexes, so the {6} doesn't work afaik. The [0-9] appears to be fine though.